extensions/ql-vscode/src/view/model-editor/__tests__/MethodRow.spec.tsx

```diff
@@ -251,4 +251,111 @@ describe(MethodRow.name, () => {
     expect(screen.queryByRole("combobox")).not.toBeInTheDocument();
     expect(screen.getByText("Method already modeled")).toBeInTheDocument();
   });
+
+  it("Doesn't show add/remove buttons when multiple methods feature flag is disabled", async () => {
```

Comment:
Most of our other tests don't start with capitals, so could we also remove those here? I think the intention is that the `it` starts the sentence, so that you'd have "it doesn't show add/remove buttons when multiple methods feature flag is disabled".